libsolidity/interface/FileReader.cpp

```diff
@@ -87,15 +107,41 @@ ReadCallback::Result FileReader::readFile(string const& _kind, string const& _so
 		if (strippedSourceUnitName.find("file://") == 0)
 			strippedSourceUnitName.erase(0, 7);
 
-		auto canonicalPath = normalizeCLIPathForVFS(
-			m_basePath / strippedSourceUnitName,
-			true /* resolveSymlinks */
-		);
+		vector<boost::filesystem::path> candidates;
+		vector<boost::filesystem::path> prefixes = {m_basePath};
+		prefixes += m_includePaths;
+
+		for (boost::filesystem::path const& prefix: prefixes)
+		{
+			boost::filesystem::path canonicalPath = normalizeCLIPathForVFS(
+				prefix / strippedSourceUnitName,
+				true /* resolveSymlinks */
+			);
+
+			if (boost::filesystem::exists(canonicalPath))
+				candidates.push_back(canonicalPath);
```

Comment:
Probably. It's a local variable but compiler might not move it automatically like it does with return values. Adding `move` won't hurt.

I tried to also make `candidates` a vector of references but I can't because `canonicalPath` is a new object, created based on the original path and prefix. But I managed to replace `prefixes` with a vector of references instead.